examples/discovery-search-app/src/ExampleDocumentProvider.js

```diff
@@ -10,12 +10,31 @@
  */
 export class ExampleDocumentProvider {
   async provides(document: DocumentProviderProps) {
-    return document.extracted_metadata.filename === 'Art Effects.pdf';
+    // return document.extracted_metadata.filename === 'Art Effects.pdf';
+    console.log('dorian change', document.extracted_metadata.filename);
+    if (document.extracted_metadata.filename.indexOf('.pdf') >= 0) {
+      // hack to force PDF to render
+      document.extracted_metadata.text_mappings = true;
```

Comment:
This seems like a bad practice. We shouldn't be modifying the object getting passed into this provider, because there might be other components or methods that assume this object is coming straight from the api response. If this started affecting the outcome of a completely different component in the future, it would be very hard to debug. We should try to fix how our code is reacting to bad data, or fix the example data at the example file we're feeding in.

Also, according to the typings we have in tooling, the `text_mappings` variable is either a string or an array of strings, but not a boolean.

Reply:
Good design point. Original file is restored.